src/librustc_mir/build/scope.rs

```diff
@@ -107,6 +107,9 @@ pub struct Scope<'tcx> {
     /// `ScopeAuxiliary`, but kept here for convenience
     extent: CodeExtent,
 
+    /// Whether there's anything for the cleanup path.
```

Comment:
I don't really know what this comment means; it doesn't seem to add much to the name. I *think* that you are distinguishing "storage-dead" from "destructor"? Or maybe something to do with panics? Anyway, I'll probably figure it out as I read, but I think you should expand it a bit. =)